web/hosts.go

```diff
@@ -15,6 +15,28 @@ import (
 	"github.com/trento-project/trento/internal/sapsystem"
 )
 
+type HealthContainer struct {
+	Passing  int
+	Warning  int
+	Critical int
+	Layout   string
+}
+
+func NewHealthContainer(hostList hosts.HostList) *HealthContainer {
+	h := &HealthContainer{}
+	for _, host := range hostList {
+		switch host.Health() {
+		case "passing":
+			h.Passing += 1
+		case "warning":
+			h.Warning += 1
+		case "critical":
+			h.Critical += 1
+		}
```

Comment:
I think we have `maintenance` status too, in the previous template logic it was squashed to the critical state

Reply:
Yes, true. But we don't use this state at all. If we start using it sometime, we will need to include it